### PR TITLE
add manual integration tests GH workflow

### PR DESCRIPTION
**Pull request by MSalopek**
# Description

Add `nightly` test run which includes all integration tests and executes them using a cronjob. The test will be performed every night at 03:00 (AM).

The `nightly` test run will notify a selected slack channel via a webhook configured for this repo.

Add manual trigger for integration tests running on `main` branch.

## Linked issues

Closes: #514 #631 
Depends on: #659 

## Type of change

If you've checked more than one of the first three boxes, consider splitting this PR into multiple PRs!

- [ ] `Feature`: Changes and/or adds code behavior, irrelevant to bug fixes
- [ ] `Fix`: Changes and/or adds code behavior, specifically to fix a bug
- [x] `Refactor`: Changes existing code style, naming, structure, etc.
- [x] `Testing`: Adds testing
- [ ] `Docs`: Adds documentation

### CODE REVIEW

File: .github/workflows/manual-integration.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          go-version: '1.18'
+      - uses: actions/checkout@v3
+
+      - name: Checkout LFS objects
+        run: git lfs checkout
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.18.0" # The Go version to download (if necessary) and use.
```
Comment (glnro) [inline .github/workflows/manual-integration.yml:23]: Why not just 1.18?

**Review by glnro — APPROVED** [review]
Looks good just one question

File: .github/workflows/nightly-integration.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Run integration tests nightly on main
+
+# !! Relevant changes to this file should be propagated manual-integration.yml
+
+name: nightly-integration-main
+on:
+  schedule:
+    # run every day at 03:00 (AM)
```
Comment (sainoe) [inline .github/workflows/nightly-integration.yml:8]: Is it 3AM in UTC?

File: .github/workflows/manual-integration.yml (hunk 1 above)
Comment (MSalopek) [reply .github/workflows/manual-integration.yml]: Accidental copy pasta. Changed!

File: .github/workflows/nightly-integration.yml (hunk 2 above)
Comment (MSalopek) [reply .github/workflows/nightly-integration.yml]: Correct, it is UTC. Added it to the comment

**Review by sainoe — APPROVED** [review]

**Pull request merged**